src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs

```diff
@@ -337,7 +337,9 @@ private void BuildParameters()
                         requestConditionHeaders |= header;
                         requestConditionRequestParameter ??= operationParameter;
                         requestConditionSerializationFormat = requestConditionSerializationFormat == SerializationFormat.Default
+                            ? (operationParameter.SerializationFormat == SerializationFormat.Default
                             ? SerializationBuilder.GetSerializationFormat(operationParameter.Type)
+                            : operationParameter.SerializationFormat)
                             : requestConditionSerializationFormat;
```

Comment:
we have a double "a ? b : c" expression here, which makes this really hard to read. Could we convert this to a method?